### PR TITLE
Minor fixes for the android sample app and add .gitignore

### PR DESCRIPTION
**Pull request by flache**
fixes are described by line comments (in the "Files changed" tab)

### CODE REVIEW

File: buildAndroid.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ export LDFLAGS="-fPIE -pie -fexceptions -Wl,-rpath-link=${NDK}/platforms/${platf
 export LDFLAGS_CORE="-fPIE -pie -fexceptions -L${NDK}/platforms/${platform}/arch-arm/usr/lib -L${NDK}/sources/cxx-stl/gnu-libstdc++/${STL_VERSION}/libs/armeabi"
 export SYSROOT=${NDK}/platforms/${platform}/arch-arm
 export CXXFLAGS="-I${NDK}/platforms/${platform}/arch-arm/usr/include -I${NDK}/sources/cxx-stl/gnu-libstdc++/${STL_VERSION}/include -I${NDK}/sources/cxx-stl/gnu-libstdc++/${STL_VERSION}/libs/armeabi/include --sysroot=${SYSROOT} -g -O3 $OPTIMISE"
-DEFINES="-DOPENSMILE_BUILD -DBUILD_LIBSVM -DBUILD_SVMSMO -DBUILD_RNNFAST -DBUILD_WITHOUT_EXPERIMENTAL"
+DEFINES="-DOPENSMILE_BUILD -DBUILD_LIBSVM -DBUILD_SVMSMO -DBUILD_RNN -DBUILD_WITHOUT_EXPERIMENTAL"
```
Comment (flache) [inline buildAndroid.sh]: the BUILD_RNNFAST flag seems to be from a previous version and has no effect. I added BUILD_RNN sine I use the LSTM VAD component. The flag is active in buildStandalone.sh so I thought I can add it as well in buildAndroid (the VAD now also works on android after the modifications in rnn.cpp below)

File: progsrc/android-template/app/src/main/jni/Android.mk [hunk 2]
```diff
@@ -11,10 +11,8 @@ include $(PREBUILT_STATIC_LIBRARY)
 
 include $(CLEAR_VARS)
 
-############ you might need to change this if you move the
-############ android studio project somewhere else, i.e.
-############ outside of the opensmile root folder
-OPENSMILE_ROOT := ../../../../../../
+############ set here the absolute path of your opensmile directory
+OPENSMILE_ROOT := /path/to/opensmile/root/dir
```
Comment (flache) [inline progsrc/android-template/app/src/main/jni/Android.mk]: It successfully run the buildAndroid.sh (`sh buildAndroid.sh`) script using the relative path, but I got a linker error when I tried build the actual android app using gradle (with ` ./gradlew assembleDebug') at the ndk compilation step:

```
make: Entering directory `OPENSMILE_ROOT/progsrc/android-template/app/src/main'
[armeabi] Gdbserver      : [arm-linux-androideabi-4.8] libs/armeabi/gdbserver
[armeabi] Gdbsetup       : libs/armeabi/gdb.setup
[armeabi] Compile++ thumb: smile_jni <= smile_jni.cpp
In file included from jni/smile_jni.cpp:51:0:
jni/smilextract.h:49:32: fatal error: core/smileCommon.hpp: No such file or directory
 #include <core/smileCommon.hpp>
                                ^
compilation terminated.
make: *** [obj/local/armeabi/objs-debug/smile_jni/smile_jni.o] Error 1
make: Leaving directory `OPENSMILE_ROOT/progsrc/android-template/app/src/main'
:app:ndkBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:ndkBuild'.
> Process 'command 'NDK_ROOT/ndk-r10e/ndk-build'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 10.148 secs
```

it went away when I was using the absolute path (and rerun `buildAndroid.sh`)

File: progsrc/android-template/app/src/main/scala/com/audeering/opensmile/androidtemplate/plugins/Config.scala [hunk 3]
```diff
@@ -26,10 +26,8 @@ object Config {
   val assets = List(
     "liveinput_android.conf",
     "BufferModeRb.conf.inc",
-    "BufferModeLive.conf.inc",
     "messages.conf.inc",
-    "music_features.conf.inc",
-    "FrameModeMoodWin.conf.inc"
+    "features.conf.inc"
```
Comment (flache) [inline progsrc/android-template/app/src/main/scala/com/audeering/opensmile/androidtemplate/plugins/Config.scala]: Android runtime error (over logcat): couldn't find the files `BufferModeLive.conf.inc`, `music_features.conf.inc`, `FrameModeMoodWin.conf.inc`. Also, the `features.conf.inc` file was not used anywhere. The app works as expected after the above modification

File: progsrc/android-template/build.gradle [hunk 4]
```diff
@@ -13,6 +13,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url 'https://maven.google.com' }
```
Comment (flache) [inline progsrc/android-template/build.gradle]: Couldn't find some dependencies when first runnig `gradlew` via command line. adding the google repository fixed the problem (found the solution on some stackoverflow post)

**Comment by naxingyu** [comment]
Thanks! I'll merge and wait for others to validate.

**Pull request merged**